assets/base.css

```diff
@@ -1509,21 +1522,41 @@ details[open] > .share-button__fallback {
   -webkit-appearance: none;
   appearance: none;
   background-color: rgb(var(--color-background));
-  border: 0.1rem solid transparent;
-  border-radius: var(--inputs-radius);
   color: rgb(var(--color-foreground));
   font-size: 1.6rem;
   width: 100%;
-  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
-  height: 4.5rem;
   box-sizing: border-box;
   transition: box-shadow var(--duration-short) ease;
-  filter: drop-shadow(
-    var(--inputs-shadow-horizontal-offset)
-    var(--inputs-shadow-vertical-offset)
-    var(--inputs-shadow-blur-radius)
-    rgba(var(--color-foreground), var(--inputs-shadow-opacity))
-  );
+  border-radius: calc((var(--inputs-border-width)) + (var(--inputs-radius)));
+  min-height: calc(4rem + (var(--inputs-border-width)) * 2);
+  min-width: calc(7rem + (var(--inputs-border-width)) * 2);
+  position: relative;
+  border: 0;
+}
+
+.field:before,
+.select:before,
+.customer .field:before,
+.customer select:before {
+  pointer-events: none;
+  content: '';
+  position: absolute;
+  inset: 0;
+  border-radius: calc((var(--inputs-border-width)) + (var(--inputs-radius)));
+  box-shadow: var(--inputs-shadow-horizontal-offset) var(--inputs-shadow-vertical-offset) var(--inputs-shadow-blur-radius) rgba(var(--color-base-text), var(--inputs-shadow-opacity));
+}
```

Comment:
This is being applied on the sort input for facet filter but i don't think it should: 
![](https://screenshot.click/22-01-8c54v-3yf01.png)
![](https://screenshot.click/22-01-iwf6y-47zkw.png)

So it probably need an overwrite. I had done one but since things have a changed a bit it's not doing anything anymore I believe.

Reply:
We decided not to style those select because of the relationship with the filters not being styled.